modules/fts/pages/fts-search-response.adoc

```diff
@@ -1,14 +1,1316 @@
+
+=== Pagination tips and recommendations
+
+The pagination of search results can be done using the 'from' and 'size' parameters in the search request. But as the search gets into deeper pages, it starts consuming more resources.  
+
+To safeguard against any arbitrary higher memory requirements, FTS provides a configurable limit bleveMaxResultWindow (10000 default) on the maximum allowable page offsets. However, bumping this limit to higher levels is not a scalable solution.
+
+To circumvent this problem, the concept of key set pagination in FTS, is introduced. 
+
+Instead of providing _from_ as a number of search results to skip, the user will provide the sort value of a previously seen search result (usually, the last result shown on the current page).  The idea is that to show the next page of the results, we just want the top N results of that sort after the last result from the previous page.
+
+This solution requires a few preconditions be met:
+
+* The search request must specify a sort order.
+
+NOTE: The sort order must impose a total order on the results.  Without this, any results which share the same sort value might be left out when handling the page navigation boundaries.  
+
+A common solution to this is to always include the document ID as the final sort criteria.                                       
+
+For example, if you want to sort by [“name”, “-age”], instead of sort by [“name”, “-age”, "_id”].
+
+
+With `search_after`/`search_before` paginations, the heap memory requirement of deeper page searches is made proportional to the requested page size alone. So it reduces the heap memory requirement of deeper page searches significantly down from the offset+from values.
+
+== Fields
+
+You can search multi-Collection indexes using the same old search requests. Since a multi-Collection index contains data from multiple source Collections, it’s useful to know the source Collection of their relevant hits.
+
+With multi-Collection indexes, each hit in the search results contains information about the Collection to which it belongs. This source Collection detail is available in the Fields section of each hit under the key _$c. See the image below for an example.
+
+image::fts-search-response-fields.png[,850,align=left]
+
+multi-Collection index search results in Couchbase 7.0
+
+You can also narrow your full-text search requests to only specific Collection(s) within the multi-Collection index. This focus speeds up your searches on a large index.
+
+== Sorting Query Results
+
+[abstract]
+The FTS results are returned as objects. FTS query includes options to order the results.
+
+=== Sorting Result Data
+
+FTS sorting is sorted by descending order of relevance.  It can , however, be customized to sort by different fields, depending on the application. 
```

Comment:
```suggestion
FTS sorting is sorted by descending order of relevance.  It can, however, be customized to sort by different fields, depending on the application. 
```

Reply:
Odd that I've seen this in an earlier document, so it looks like it might have been copied and pasted. 
If you have common elements in a document, then it's best to abstract them out into an include file:

https://docs.asciidoctor.org/asciidoc/latest/directives/include/

Reply:
updated